.github/workflows/ci.yaml

```diff
@@ -18,6 +18,18 @@ jobs:
           path: ~/.cache/pip
           key: ${{ hashFiles('pyproject.toml') }}
       - uses: pre-commit/action@v2.0.3
+
+  Type-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
```

Comment:
Should we change this to Python 3.9 and consider it as default until Airflow releases a stable version that supports 3.10 officially - this will happen in Airflow 2.3, wdyt?